openedx/core/djangoapps/notifications/templates/notifications/edx_ace/email_digest/email/subject.txt

```diff
@@ -0,0 +1 @@
+{{ digest_frequency }} Notifications Digest for {% if digest_frequency == "Weekly" %}the Week of {% endif %}{{ start_date }}
```

Comment:
What happens here for `Daily` Digest?

Reply:
For daily digest: `Daily Notifications Digest for Jun 12`
For weekly digest: `Weekly Notifications Digest for the week of Jun 12`